src/memory-analyzer/analyze_symbol.h

```diff
@@ -0,0 +1,114 @@
+// Copyright 2018 Author: Malte Mues
+#ifdef __linux__
+#ifndef CPROVER_MEMORY_ANALYZER_ANALYZE_SYMBOL_H
+#define CPROVER_MEMORY_ANALYZER_ANALYZE_SYMBOL_H
+#include <map>
+#include <string>
+
+#include <ansi-c/expr2c_class.h>
+#include <util/message.h>
+#include <util/namespace.h>
+#include <util/std_code.h>
+
+class symbol_tablet;
+class gdb_apit;
+class exprt;
+class source_locationt;
+
+class symbol_analyzert : public messaget
```

Comment:
Probably shouldn't inherit from `messaget`? `symbol_analyzert` isn't *a* logger, it *has* a logger.

Reply:
@chrisr-diffblue I addressed the comments which weren't fixed already in https://github.com/danpoe/cbmc/pull/7.